repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -1263,14 +1263,15 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
         |    </user>
         |    <serviceCatalog/>
         |    <RAX-AUTH:extendedAttributes xmlns="http://docs.rackspace.com/identity/api/ext/RAX-AUTH/v1.0"
-        |                                 xmlns:RAX-AUTH="http://docs.rackspace.com/identity/api/ext/RAX-AUTH/v1.0">
+        |                                 xmlns:RAX-AUTH="http://docs.rackspace.com/identity/api/ext/RAX-AUTH/v1.0"
+        |                                 xmlns:mapping="http://docs.rackspace.com/identity/api/ext/MappingRules">
         |        <group name="user">
         |            <attribute name="foo">
         |                <value>2017-01-25T21:50:56.399-06:00</value>
         |            </attribute>
         |        </group>
         |        <group name="test">
-        |            <attribute name="policy">
+        |            <attribute multiValue="true" name="policy">
```

Comment:
I find this kind of odd, but apparently this is the output we are getting.

I would have expected this attribute to be namespaced with `mapping`.

Reply:
Nope, this is covered in the extended attributes extension, which we control.  The mapping: namespace is used within SAML to avoid conflicts in the SAML response.

Reply:
Fair enough. However, it appears that namespaces from the SAML response are pulled into the access response? I'll point it out above.